spring-restdocs-core/src/main/java/org/springframework/restdocs/cli/CurlRequestSnippet.java

```diff
@@ -45,6 +44,8 @@
  */
 public class CurlRequestSnippet extends TemplatedSnippet {
 
+	private CommandFormatter commandFormatter = ConcatenatingCommandFormatter.newlineSeparator();
+
 	/**
 	 * Creates a new {@code CurlRequestSnippet} with no additional attributes.
 	 */
```

Comment:
This constructor should be deprecated and a new constructor that takes a `CommandFormatter` introduced